app-modules/contact/tests/Organization/CreateOrganizationTest.php

```diff
@@ -80,6 +80,14 @@
 
     livewire(CreateOrganization::class)
         ->fillForm($request->toArray())
+        ->set('data.domains', [
+            [
+                'domain' => 'brakus.com',
+            ],
+            [
+                'domain' => 'purdy.com',
+            ],
```

Comment:
Is there some way to flatten this structure in Filament so that it is just an array of the domain names without the `domain` key?

Reply:
Hi @Orrison 
With flatten it's not possible as the repeater is only accepting key-value pair array.